2019/MDAnalysis_ideas_list.md

```diff
@@ -1 +1,84 @@
+
+__repository:__ https://github.com/MDAnalysis/mdanalysis
+
+__email:__ mdnalysis-devel@googlegroups.com
+
+__Project name:__ Make it easy for new users to analyze _their_ data  
+
+__Description:__ MDAnalysis is Python library that provides an abstract and object-oriented interface to data from particle-based simulations (primarily molecular dynamics simulations), which are widely used for simulating diverse systems such as the interaction of drugs with biomolecules or new materials. MDAnalysis is widely used in the scientific community and is written by scientists for scientists. Feedback from our users indicates that they like using MDAnalysis but wished that the documentation were easier to read and had more examples. The docs for [scikit-learn](https://scikit-learn.org/stable/index.html) and the [PyTorch tutorials](https://pytorch.org/tutorials/index.html) are generally cited as excellent examples of documentation and, taking these as examples, we would like to improve our documentation to make it more accessible and more immediately useful for new users. 
+
+We identified four areas for improvement (in rank order of priority):
+
+### Restructure docs
+
+We want to **restructure our docs for user-friendliness** [issue #1175](https://github.com/MDAnalysis/mdanalysis/issues/1175) and refactor docs away from how the source code is organized into how the user interacts with the code (started in [PR #1827](https://github.com/MDAnalysis/mdanalysis/pull/1827)). We envisage a split into three major blocks:
+
+1. introduction with examples (more like a tutorial) and explanation of the underlying principles and guiding concepts (see the [SciPy 2016 talk](https://www.mdanalysis.org/pages/learning_MDAnalysis/#introductory) and the slides in the presentation [scipy-MDAnalysis-Beckstein.pdf](https://github.com/MDAnalysis/scipy-2016/blob/master/presentation/scipy-MDAnalysis-Beckstein.pdf))
+2. API docs (similar to the current docs at https://www.mdanalysis.org/docs/)
+3. developer docs (notes for developers, can be technical/arcane)
+
+### Improve and expand tutorials
+
+We have one "official" [introductory tutorial](https://www.mdanalysis.org/MDAnalysisTutorial/) and [various other tutorials](https://www.mdanalysis.org/pages/learning_MDAnalysis/#tutorials) but it is initially confusing to new users what they should look at and it is too long. We need to provide a better "road map" for new users and clearly lay out tutorials for different levels and with clear learning goals. 
+
+We need to split the current [MDAnalysis Tutorial](https://www.mdanalysis.org/MDAnalysisTutorial/) into multiple self-contained tutorials and sort them by level (introductory, intermediate, advanced). The tutorials can and should build on each other. There should be a top level entry point that gives an overview over the tutorials. An initial outline would contain the following (not all content exists yet, especially at intermediate/advanced):
```

Comment:
good to point to the paper for this as well. The introduction into mdanalysis is better in the papers than in the actual documentation. Is there another md library with good docs that could be shown as an example?

Reply:
Very good comment, will include.